README.md

```diff
@@ -106,15 +107,82 @@ example with [fatih/color](https://github.com/fatih/color):
 ```
 ### adaptors
 
-`lgr` logger can be converted to `io.Writer` or `*log.Logger`
+`lgr` logger can be converted to `io.Writer`, `*log.Logger`, or `slog.Handler`
 
 - `lgr.ToWriter(l lgr.L, level string) io.Writer` - makes io.Writer forwarding write ops to underlying `lgr.L`
 - `lgr.ToStdLogger(l lgr.L, level string) *log.Logger` - makes standard logger on top of `lgr.L`
+- `lgr.ToSlogHandler(l lgr.L) slog.Handler` - converts lgr.L to a slog.Handler for use with slog
 
 _`level` parameter is optional, if defined (non-empty) will enforce the level._
 
 - `lgr.SetupStdLogger(opts ...Option)` initializes std global logger (`log.std`) with lgr logger and given options. 
 All standard methods like `log.Print`, `log.Println`, `log.Fatal` and so on will be forwarder to lgr.
+- `lgr.SetupWithSlog(logger *slog.Logger)` sets up the global logger with a slog logger.
+
+### slog integration
+
+In addition to the standard logger interface, lgr provides seamless integration with Go's `log/slog` package:
+
+#### Using lgr with slog
+
+```go
+// Create lgr logger
+lgrLogger := lgr.New(lgr.Debug, lgr.Msec)
+
+// Convert to slog handler and create slog logger
+handler := lgr.ToSlogHandler(lgrLogger)
+logger := slog.New(handler)
+
+// Use standard slog API with lgr formatting
+logger.Info("message", "key1", "value1")
+// Output: 2023/09/15 10:34:56.789 INFO  message key1="value1" 
+```
+
+#### Using slog with lgr interface
+
+```go
+// Create slog handler
+jsonHandler := slog.NewJSONHandler(os.Stdout, nil)
+
+// Wrap it with lgr interface
+logger := lgr.FromSlogHandler(jsonHandler)
+
+// Use lgr API with slog backend
+logger.Logf("INFO message with %s", "structured data")
+// Output: {"time":"2023-09-15T10:34:56.789Z","level":"INFO","msg":"message with structured data"}
+```
+
+#### Using slog directly in lgr
+
+```go
+// Create a logger that uses slog directly
+jsonHandler := slog.NewJSONHandler(os.Stdout, nil)
+logger := lgr.New(lgr.SlogHandler(jsonHandler))
+
+// Use lgr API with slog backend
+logger.Logf("INFO message")
+// Output: {"time":"2023-09-15T10:34:56.789Z","level":"INFO","msg":"message"}
+```
+
+#### JSON output with caller information
+
+To get caller information in JSON output when using slog handlers, create the handler with `AddSource: true`:
+
+```go
+// Create JSON handler with source information (caller info)
+jsonHandler := slog.NewJSONHandler(os.Stdout, &slog.HandlerOptions{
+    AddSource: true,  // This enables caller information in JSON output
+})
+
+// Use handler with lgr
+logger := lgr.New(lgr.SlogHandler(jsonHandler))
+
+logger.Logf("INFO message with caller info")
+// Output will include source file, line and function in JSON
+```
+
+Note: The lgr caller options (`lgr.CallerFile`, `lgr.CallerFunc`) only work with lgr's native text format
```

Comment:
I propose to add comments to CallerFile and CallerFunc to highlight that they won't affect slog output in case slog handler is used.